components/datepicker/stories/datepicker.stories.js

```diff
@@ -149,3 +149,108 @@ Range.args = {
 	isRange: true,
 	isOpen: false,
 };
+
+/**
+ * Stories for the MDX "Docs" only.
+ */
+export const QuietRange = Template.bind({});
+QuietRange.args = {
+	month: "March",
+	selectedDay: 1,
+	year: 2023,
+	lastDay: 3,
+	content: [{}],
+	isRange: true,
+	isQuiet: true,
+	isOpen: false,
+};
+QuietRange.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+QuietRange.tags = ["is-hidden-story"];
+
+export const ReadOnly = Template.bind({});
+ReadOnly.args = {
+	readOnly: true,
+	month: "March",
+	selectedDay: 1,
+	year: 2023,
+	content: [{}],
+};
+ReadOnly.parameters = {
+	chromatic: { disableSnapshot: true },
+	docs: {
+		story: {
+			height: "60px",
+		}
+	}
+};
```

Comment:
I found some vertical scrolling with the `ReadOnly`, `Disabled` and `QuietDisabled` stories, I assume because of these `height: 60px`s. Was that intentional? I just wasn't expecting it.

https://github.com/adobe/spectrum-css/assets/69602589/e8c863ff-7f58-4e77-a6f5-4183356d64db

Reply:
Ah yep, because even if it's disabled that calendar code is still rendering... I'm going to remove the calendar code for those variants. Good catch.